bundle/Resources/config/ezplatform.yml

```diff
@@ -1,4 +1,7 @@
 system:
+    admin_group:
+        field_templates:
+            - {template: "@NetgenTags/admin/eztags_content_field.html.twig", priority: 1}
```

Comment:
I would use the priority of 0, so it is easier to overwrite with user defined priorities.

Reply:
The override does not work for me with a priority of 0 (I don't understand why, by the way). Then this whole override to distinct admin/global display becomes useless.

Reply:
Hm, that sounds weird. I'll try to reproduce it also.

Reply:
Indeed, this does not work with the priority of `0`. I'm not sure why, but I have a hunch it might be because of https://github.com/ezsystems/ezplatform-standard-design/pull/2#pullrequestreview-123300958

Anyway, the priority of `1` can obviously stay.